sql.txt

```diff
@@ -1,14 +1,29 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country FROM [Customers] ORDER BY Country LIMIT 10 OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM [Customers] WHERE ContactName LIKE 'A%';
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * FROM [OrderDetails] WHERE ProductID = 51 AND Quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products VALUES 
+(NULL, "product1", 26, 5, "1 box", 5),
+(NULL, "product2", 28, 6, "1 box", 5),
+(NULL, "product3", 30, 7, "1 box", 5)
+
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+UPDATE products SET price = 50 WHERE ProductID IN (1,2);
```

Comment:
You are supposed to use math here to get the price. So it should be the current price + 1.50. This needs an edit to meet what was asked.